CMakeLists.txt

```diff
@@ -261,14 +261,7 @@ if(USE_OPENCV)
   message(STATUS "OpenCV found (${OpenCV_CONFIG_PATH})")
   add_definitions(-DMXNET_USE_OPENCV=1)
   if(NOT MSVC)
```

Comment:
is there anything left in this if(NOT MSVC)/else() block after your change?  it doesn't look like it.

Reply:
You said "don't remove this block". Happy to remove it.